### ISSUE TITLE
Breaks with Language Selector

### ISSUE THREAD

**Issue by andrewfenn** [bleutzinn/grav-plugin-editable-contenttools#3]
This plugin doesn't seem to work well with the [Language Selector](https://github.com/clemdesign/grav-plugin-language-selector) plugin. When I switch to a language then edit the contents, click save it reverts to the previous content.

**Comment by bleutzinn** [comment]
Did you insert the shortcodes in all language versions ?

**Comment by andrewfenn** [comment]
You are correct. I was using just on one language so I went back and tested. In my pages I have the following code with different content for each Language for testing..

```
[editable name="region-0"]
English
[/editable]
```

When I go to the chinese page (http://grav.local/zh-tw/) and edit the content and hit save the UI looks as expected, but the content is changed on the default english page (http://grav.local/). It does not update on the pages language I edited the code on.

I next tried to change the name for each editable tag in each language...

English Page
```
[editable name="region-home-en"]
English
[/editable]
```

Chinese Page - hans
```
[editable name="region-home-zh"]
Chinese
[/editable]
```

Chinese Page - hant
```
[editable name="region-home-zh-tw"]
Chinese TW
[/editable]
```

When I use editable content to edit the (http://grav.local/zh-tw/) the UI works but nothing is saved. Here are some screenshots to show I am using just one page with different language settings.

![screenshot from 2019-02-28 11-52-00](https://user-images.githubusercontent.com/409265/53542070-4f46b580-3b4f-11e9-868b-86dc56584268.png)
![screenshot from 2019-02-28 11-51-48](https://user-images.githubusercontent.com/409265/53542071-4f46b580-3b4f-11e9-8626-7750d21a0905.png)

**Comment by bleutzinn** [comment]
Thanks for the extra info. I'll try to setup a multi language site and see where things go wrong.

**Comment by bleutzinn** [comment]
Should be fixed in version 1.5.0.

Instead of testing with the Language Selector Plugin, I tested with the [MultiLang Skeleton](https://github.com/getgrav/grav-skeleton-multilang-site) which uses the [LangSwitcher Plugin](https://github.com/getgrav/grav-plugin-langswitcher) out of the box.

As far as I can tell both these language selecting plugins only differ in their user interface.

BTW there is no need to set the name for each editable tag in each language page.

**Comment by andrewfenn** [comment]
Thanks for the feedback and advice.
